### ISSUE TITLE
Pure samples

### ISSUE THREAD

**Issue by eracah** [NERSC/dayabay-learn#23]
Hey Sam, do you have any pure samples. That is, events that you and other physicistsare almost certain are IBD events. That might be useful for semi-supervised learning or for coloring some of tsne points

**Comment by samkohn** [comment]
I can get closer than what we have now (94% IBD, 5% Lithium-9, and 1% accidentals). It might take a few days. I could probably get up to 97% or 98% purity.

**Comment by eracah** [comment]
Ok. Just curious. Because we were talking about doing semi-supervised. You think 98% is good enough? 
I think we could do an anomaly detection method
or
noisy label semi-supervised method?

**Comment by eracah** [comment]
I think anomaly detection works with polluted samples

**Comment by samkohn** [comment]
Unfortunately I'm not sure how good is good enough. My experience so far has been that the DCA reconstructs well on signal and poorly on the 1% of events that are accidental background. This was why I am working to get an intentionally-contaminated sample. That experience leads me to think that 98% purity would be sufficient. But it's just a guess.

I read up on the adversarial and variational bayes NNs. It turns out I know one of the authors on the original generative adversarial nets paper.

**Comment by samkohn** [comment]
I was mistaken about the Lithium-9 background contamination. The correct breakdown for our current sample is approximately:

Lithium-9: 0.4% (less that what I remembered by a factor of 10)
Accidentals: 1% (unchanged from what I told you earlier)
True IBD: 98.6%

As I mentioned, I'm working to get an unbiased sample of only accidental backgrounds. I will also start generating a sample enhanced significantly in Lithium-9 compared to the 0.4% that we currently have.

I know you're also hoping for a higher-purity signal set, but I talked to Dan and we haven't figured out yet how to increase the purity without significantly biasing the remaining sample.

**Comment by eracah** [comment]
Hmmm. Well really pure is obviously nice, but a little pollution is ok. 0.4 % is pretty small. 

I didn't realize that it was so small. Is this still an important problem for physicists if its that small?

**Comment by samkohn** [comment]
Yes, it is: our confidence in the value of that number (0.4%) is relatively low—we think it is 0.4% +/- 0.2%. The goal is to increase our confidence in how well we can estimate that contamination.
